clang/include/clang/Basic/Attr.td

```diff
@@ -592,6 +592,46 @@ class AttrSubjectMatcherAggregateRule<AttrSubject subject> {
 
 def SubjectMatcherForNamed : AttrSubjectMatcherAggregateRule<Named>;
 
+// Late Attribute parsing mode enum
```

Comment:
@rapidsna I renamed the `LateAttrParseKind` definitions and updated the comments explaining the semantics. Does this look right to you?

Reply:
@delcypher Thanks! LGTM.